zstacklib/zstacklib/utils/sanlock.py

```
@@ -121,9 +129,14 @@ def get_lockspace_record(self, needle):
         return None
 
-    @linux.retry(3, 1)
-    def _get_sanlock_status(self):
-        return bash.bash_errorout("sanlock client status -D")
+    def _init(self):
+        @linux.retry(3, 1)
+        def _get():
+            return bash.bash_errorout("timeout 10 sanlock client status -D")
+        try:
+            return _get()
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)